docs/linking.md

```diff
@@ -91,6 +80,37 @@ If your app is using [Universal Links](https://developer.apple.com/ios/universal
 }
 ```
 
+</TabItem>
+<TabItem value="swift">
+
+```swift title="AppDelegate.swift"
+// Make sure you have the `import React` statement at the top of the file
+
+func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any] = [:]) -> Bool {
+  return RCTLinkingManager.application(app, open: url, options: options)
+}
+```
+
+If your app is using [Universal Links](https://developer.apple.com/ios/universal-links/), you'll need to add the following code as well:
+
+```swift title="AppDelegate.swift"
+// Make sure you have the `import React` statement at the top of the file
```

Comment:
Q: Why can't we just have an import React here instead of having a comment?

Reply:
because it should be already there. I just add a comment to let our users know that it should be there, but in 99% of the cases it's there because the template is like this: https://github.com/react-native-community/template/blob/main/template/ios/HelloWorld/AppDelegate.swift#L2

Reply:
> Do you think I should just remove the comment?

Probably yes. Is like the package/import statement in Kotlin. We add them only if they could be confusing for the user in that specific context, otherwise we omit it